auth/credentials/internal/externalaccount/url_provider.go

```diff
@@ -30,6 +30,7 @@ const (
 	fileTypeJSON             = "json"
 	urlProviderType          = "url"
 	programmaticProviderType = "programmatic"
+	x509ProviderType         = "x509"
```

Comment:
side comment: I don't think url_provider is the right location to define the list of provider types.

Reply:
I can fix this up in a future PR, not sure why this landed here 😄